DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/Index.cshtml

```diff
@@ -0,0 +1,114 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.Common
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
+@using Microsoft.Extensions.Configuration
+@model ViewDelegateViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/viewDelegate.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "View delegate";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+
+  var activeTagCss = Model.DelegateInfo.IsActive ? "nhsuk-tag nhsuk-tag--green" : "nhsuk-tag nhsuk-tag--red";
+  var passwordTagCss = Model.DelegateInfo.IsPasswordSet ? "nhsuk-tag nhsuk-tag--green" : "nhsuk-tag nhsuk-tag--red";
+
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+@section NavBreadcrumbs {
+  <partial name="~/Views/TrackingSystem/Delegates/Shared/_AllDelegatesBreadcrumbs.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-two-thirds">
+        <h1 class="nhsuk-heading-xl nhsuk-u-margin-bottom-3">@Model.DelegateInfo.Name</h1>
+      </div>
+      @if (Model.DelegateInfo.IsActive) {
+        <div class="nhsuk-grid-column-one-third view-delegate-top-button-group">
+          <a class="nhsuk-button nhsuk-button--secondary view-delegate-button " role="button">
+            Send Welcome Email
+          </a>
+        </div>
+      }
+    </div>
+
+    <div class="nhsuk-u-margin-bottom-5">
+      <strong class="@activeTagCss">@(Model.ActiveTagName)</strong>
+      <strong class="nhsuk-tag nhsuk-tag--grey">@(Model.RegStatusTagName)</strong>
+      <strong class="@passwordTagCss">@(Model.PasswordTagName)</strong>
+      @if (Model.DelegateInfo.IsAdmin) {
+        <strong class="nhsuk-tag nhsuk-tag--grey">Admin</strong>
+      }
+    </div>

```

Comment:
I don't think anything needs to be done right now, but as search/sort/etc. come into play, the SearchableDelegateViewModel will eventually implement BaseFilterableViewModel, which has its own way of dealing with tags that is designed to be consistent for filtering purposes. Its kinda unnecessary here as its just a single record display, but worth bearing in mind if the AllDelegates page changes.

Reply:
This page doesn't need the sorting/searching properties, though, does it? That's why I haven't used SearchableDelegateViewModel here.

Reply:
Yeah, this page doesn't need any searching/sorting properties. The comment was more about if we wanted to change to using the SearchableDelegateViewModel which had a slightly different way of handling tags, which we don't so its fine as is.